js/components/operatorSignIn/expiry.tsx

```diff
@@ -0,0 +1,147 @@
+import { daysBetween } from "../../dateTime";
+import {
+  Certification,
+  humanReadableType,
+  isExpired,
+} from "../../models/certification";
+import { className } from "../../util/dom";
+import { DateTime } from "luxon";
+import { ReactElement } from "react";
+
+const WARN_WITHIN_D = 60;
+
+const englishDaysBetween = (now: DateTime, date: DateTime) => {
+  const delta = daysBetween(now, date);
+  const wholeAbs = Math.abs(Math.ceil(delta));
+  if (delta <= -2) {
+    return `expired ${wholeAbs} days ago`;
+  } else if (delta <= -1) {
+    return "expired yesterday";
+  } else if (delta <= 0) {
+    return "expired today";
+  } else if (delta <= 1) {
+    return "expires tomorrow";
+  } else {
+    return `expires in ${wholeAbs} days`;
+  }
+};
+
+const CertificateBox = ({
+  now,
+  mode,
+  title,
+  operatorName,
+  certifications,
+}: {
+  now: DateTime;
+  mode: "warning" | "error";
+  title: string;
+  operatorName: string;
+  certifications: Certification[];
+}): ReactElement => {
+  const innerString =
+    `Our records show that ${operatorName}'s ` +
+    certifications
+      .map(
+        (c) =>
+          `${humanReadableType(c.type)} ${englishDaysBetween(
+            now,
+            c.expires,
+          )} on ${c.expires.toLocaleString(DateTime.DATE_SHORT)}`,
+      )
+      .join(" and ") +
+    ".";
+  return (
+    <div
+      className={className([
+        "mb-4 mt-2 flex flex-row rounded border-0 px-3 py-2",
+        mode === "warning" && "bg-[#FFDE9E]",
+        mode === "error" && "bg-[#FF919A]",
```

Comment:
A previous version (when there was the WarningParagraph component) had some extra classes. `light:bg-opacity-40 dark:bg-opacity-30 dark:text-white`. Did you mean to remove those in f54a72135dc4800702ef9b13322783a12595e4e1 ? (Also, I don't think `border-0` is needed anymore, since no border should be the default.)

Reply:
Indeed, all intentional 👍 we don't have light/dark mode and the colors are exact. thanks for catching border-0!